src/components/ScaleLattice.vue

```diff
@@ -1,19 +1,195 @@
 otonalFundamental
 <script setup lang="ts">
-import type { Scale } from "scale-workshop-core";
-import { ref } from "vue";
-
-defineProps<{
+import { access } from "fs";
```

Comment:
Huh?

Reply:
VSCode autoimport (didn't clean this up), lol